### PR TITLE
[AUTO-7448] add standard actions for v2.X tests

### PR DESCRIPTION
**Pull request by anirudhwarrier**
https://smartcontract-it.atlassian.net/browse/AUTO-7448

For Automation v2/v2.1 we have different integration-tests. 
But some like the benchmark test uses different deployment code. The aim of this PR is to have common actions that can be reused across any test.

### CODE REVIEW

File: integration-tests/contracts/contract_deployer.go [hunk 1]
```diff
@@ -763,6 +764,25 @@ func (e *EthereumContractDeployer) DeployUpkeepTranscoder() (UpkeepTranscoder, e
 	}, err
 }
 
+func (e *EthereumContractDeployer) LoadUpkeepTranscoder(address common.Address) (UpkeepTranscoder, error) {
```
Comment (FelixFan1992) [inline integration-tests/contracts/contract_deployer.go:767]: there are multiple versions of transcoder but maybe this should deploy the most recent version 4.0?

Comment (anirudhwarrier) [reply integration-tests/contracts/contract_deployer.go:767]: Good point. Created task - https://smartcontract-it.atlassian.net/browse/AUTO-7455

Comment (FelixFan1992) [reply integration-tests/contracts/contract_deployer.go:767]: just a note: 4.0 only works with upkeeps for 2.1 registry. 3.0 works with 2.0 registry. but i don't see a need to migrate to 2.0.

File: integration-tests/contracts/contract_deployer.go [hunk 2]
```diff
@@ -763,6 +764,25 @@ func (e *EthereumContractDeployer) DeployUpkeepTranscoder() (UpkeepTranscoder, e
...
+	instance, err := e.client.LoadContract("UpkeepTranscoder", address, func(
+		address common.Address,
+		backend bind.ContractBackend,
+	) (interface{}, error) {
+		return upkeep_transcoder.NewUpkeepTranscoder(address, backend)
+	})
+
```
Comment (ferglor) [inline integration-tests/contracts/contract_deployer.go:774]: 🏄‍♂️ I would drop this new line

**Review by ferglor — APPROVED** [review]

File: integration-tests/contracts/contract_deployer.go (hunk 2 above)
Comment (anirudhwarrier) [reply integration-tests/contracts/contract_deployer.go:774]: Have another task to update the upkeep transcoder version. Will clean up then.

**Pull request merged**